examples/boards/xula/xula_blinky_host.py

```diff
@@ -29,15 +29,32 @@ def xula2_blinky_host(clock, reset, led, bcm14_txd, bcm15_rxd):
     # create the interfaces to the UART
     fbustx = FIFOBus(width=8, size=4)
     fbusrx = FIFOBus(width=8, size=4)
+    uart_fifo = FIFOBus(width=8, size=4)
 
     # create the memmap (CSR) interface
     memmap = Barebone(glbl, data_width=32, address_width=32)
 
     # create the UART instance.
-    uart_inst = uartlite(glbl, fbustx, fbusrx,
+    uart_inst = uartlite(glbl, uart_fifo,
                          serial_in=bcm14_txd,
                          serial_out=bcm15_rxd)
 
+    @always_comb
+    def sync_write():
+        # sync between the UART fifo interface serial out
+        # and the command bridge out
+        uart_fifo.write_data.next = fbustx.write_data
+        uart_fifo.write.next = fbustx.write
+        fbustx.full.next = uart_fifo.full
+
+    @always_comb
+    def sync_read():
```

Comment:
it's a minor point, this isn't "sync'n" anything, we are simply assigning the signals to each other.  In myhdl we only have the `always_comb` to assign signals.  Something like `assign_write` and `assign_read` might be more appropriate.

I still think a function (method) in `FIFOBus` would be a better solution, you ended up repeating the same code in a bunch of spots (DRY).